packages/quizzes-dashboard/pages/stateless/index.tsx

```diff
@@ -104,8 +101,53 @@ const normalizeData = (data: any) => {
     questions: normalizedInputData.entities.questions ?? {},
   }
 }
+interface QuizzesSpecs {
+  private_spec: Quiz
+  public_spec: { id: string; items: PublicSpecItem[] }
+}
+
+const convertStoreToSpecs = (store: storeState): QuizzesSpecs => {
+  const denormalizedData = denormalizeData(store)
+
+  const specs: QuizzesSpecs = {
+    private_spec: denormalizeData(store),
+    public_spec: {
+      id: v4(),
```

Comment:
move to here:  https://github.com/rage/secret-project-331/pull/264/files#diff-9f749ade62c412f70c18d0f442fc664bfeac83c227353dec530baf65c0943f99R1